<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ques4</class>
 <widget class="QMainWindow" name="ques4">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1046</width>
    <height>678</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Question 4</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>300</y>
      <width>471</width>
      <height>231</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Question 4: Out of these statements which ones are TRUE?</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="R1">
       <property name="text">
        <string>Some metals are so reactive that they explode on contact with water.</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="R2">
       <property name="text">
        <string>Eating long beans make you taller</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="R3">
       <property name="text">
        <string>Not finishing the food on your plate will give you bad luck</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="R4">
       <property name="text">
        <string>Police will come if you open the courtesy lights in a car</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="B1">
       <property name="text">
        <string>Continue</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>70</y>
      <width>431</width>
      <height>111</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Name: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="T1">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Remaining Time: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="T2"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1046</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
